### PR TITLE
Test BoundingBox parsing

### PR DESCRIPTION
**Pull request by SPalominos**
Adds a test class to test the parsing of BoundingBoxData input/output. (part of the issue #1200)

### CODE REVIEW

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+                inputDescriptionType.getIdentifier().getValue());
+        Assert.assertTrue("The InputDescriptionType metadata attribute should be empty",
+                inputDescriptionType.getMetadata().isEmpty());
+    }
+
+    /**
+     * Tests the parsing of a complex BoundingBox input.
+     */
+    @Test
+    public void testCompexParseInput(){
```
Comment (ebocher) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: complex ?

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+                outputDescriptionType.getIdentifier().getValue());
+        Assert.assertTrue("The OutputDescriptionType metadata attribute should be empty",
+                outputDescriptionType.getMetadata().isEmpty());
+    }
+
+    /**
+     * Tests the parsing of a complex BoundingBox output.
+     */
+    @Test
+    public void testCompexParseOutput(){
```
Comment (ebocher) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: complex ?

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+        Assert.assertEquals("The role of the first metadata is not the one expected", "role",
+                outputDescriptionType.getMetadata().get(0).getRole());
+        Assert.assertEquals("The title of the first metadata is not the one expected", "title",
+                outputDescriptionType.getMetadata().get(0).getTitle());
+    }
+
+    /**
+     * Class used to declare and get fields with the annotation to parse.
+     */
+    private class FieldProvider{
```
Comment (ebocher) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: Could you please also test the boundingbox values ?

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java (hunk 2 above)
Comment (SPalominos) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: Fixed in the incoming commit

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java (hunk 3 above)
Comment (SPalominos) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: Fixed in the incoming commit

File: bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java (hunk 1 above)
Comment (SPalominos) [inline bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/parser/BoundingBoxParserTest.java]: Fixed in the incoming commit

File: bundles/wps/wps-service/src/main/java/org/orbisgis/wpsservice/model/BoundingBoxData.java [hunk 4]
```diff
@@ -106,6 +109,22 @@ protected BoundingBoxData(){
...
+     */
+    public void setDefaultValue(String defaultValue) {
+        this.defaultValue = defaultValue;
+    }
+
+    /**
+     * Returns the default value.
+     * @return The default value.
+     */
+    public String getDefaultValue() {
```
Comment (ebocher) [inline bundles/wps/wps-service/src/main/java/org/orbisgis/wpsservice/model/BoundingBoxData.java]: Why a default value on a BB object ?
Could you explain ?

**Pull request merged**